Bottom: 772a029ce210b4f3546d771be568888b7daa6e19
Top:    5d8e80dda4a13d5c6ac57c73862bd9b9b0124bb0
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-08 23:53:42 +0100

drc: HACK HACK HACK


---

diff --git a/src/find.c b/src/find.c
index 2f9053d..2b2104e 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2747,7 +2747,12 @@ IsPolygonInPolygon (PolygonTypePtr P1, PolygonTypePtr P2)
 
               line.Point1.X = v->point[0];
               line.Point1.Y = v->point[1];
-              line.Thickness = 2 * Bloat;
+              line.Thickness = Bloat;
+              /* Another Bloat is added by IsLineInPolygon, making the correct
+               * 2x Bloat. Ideally we would change it there, but doing so
+               * breaks some other DRC checks which rely on the broken behaviour
+               * in IsLineInPolygon.
+               */
               line.Clearance = 0;
               line.Flags = NoFlags ();
               for (v = v->next; v != &c->head; v = v->next)
